scripts/jsdeps.py

```diff
@@ -0,0 +1,39 @@
+import json
+import shutil
+from pathlib import Path
+
+repoRoot = Path(__file__).resolve().parent.parent
+destDir = repoRoot / "src/fontra/client/third-party/"
+
+
+def loadPackageDependencies():
+    with open("package.json") as packageFile:
+        packageInfo = json.load(packageFile)
+        dependencies = packageInfo.get("dependencies")
```

Comment:
This can be simplified: 
- from this line on it doesn't need to be within the `with` block
- Use `{}` as the fallback in `.get("dependencies")`, which allows you to get rid of the `if`, and the fallback return statement.